src/vivarium_nih_us_cvd/model_specifications/heart_failure_pafs.yaml

```diff
@@ -0,0 +1,58 @@
+components:
+    vivarium_public_health:
+        population:
+            - BasePopulation()
+        metrics:
+            - ResultsStratifier()
+
+    vivarium_nih_us_cvd:
+        components:
+            # heart failure
+            - IschemicHeartDiseaseAndHeartFailure()
+            # risks
+            - TruncatedRisk("risk_factor.high_systolic_blood_pressure")
+            - CategoricalSBPRisk()
+            - TruncatedRisk("risk_factor.high_body_mass_index_in_adults")
+            # risk effects
+            - PAFCalculationRiskEffect("risk_factor.categorical_high_systolic_blood_pressure", "cause.heart_failure_from_ischemic_heart_disease.incidence_rate")
+            - PAFCalculationRiskEffect("risk_factor.categorical_high_systolic_blood_pressure", "cause.heart_failure_residual.incidence_rate")
+            - PAFCalculationRiskEffect("risk_factor.high_body_mass_index_in_adults", "cause.heart_failure_from_ischemic_heart_disease.incidence_rate")
+            - PAFCalculationRiskEffect("risk_factor.high_body_mass_index_in_adults", "cause.heart_failure_residual.incidence_rate")
+
+            # PAF observers
+            - PAFObserver("risk_factor.high_body_mass_index_in_adults", "cause.heart_failure_residual.incidence_rate")
+            - PAFObserver("risk_factor.categorical_high_systolic_blood_pressure", "cause.heart_failure_residual.incidence_rate")
+
+configuration:
+    input_data:
+        input_draw_number: 0
+        location: 'Alabama'
+        artifact_path: '/home/hussain-jafari/artifacts/alabama.hdf'
+    interpolation:
+        order: 0
+        extrapolate: True
+    randomness:
+        map_size: 1_000_000
+        key_columns: ['entrance_time', 'age']
+        random_seed: 0
+    time:
+        start:
+            year: 2021 # Includes two years for burn-in
+            month: 1
+            day: 1
+        observation_start: # TODO: Implement for all observers (currently only custom observers)
+            year: 2021
+            month: 1
+            day: 1
+        end:
+            year: 2021
+            month: 1
+            day: 2
+        step_size: 28 # Days
+    population:
+        population_size: 100_000

```

Comment:
Did we get clarification from the RT what population size we want to run with? And if they want to parallelize by age group to guarantee we get enough people in each age group to calculate the PAF?

Reply:
For our first run Syl said we want 100,000 population size with 25 draws. I brought up parallelizing by age group and it's something she'll consider but for this run we can run without it. I'll ask again.